app/views/transactions/index.html.slim

```diff
@@ -1,3 +1,7 @@
+.row
+  .col-md-10
+    h5.nav.navbar-text.mr-1= t('.balance')
```

Comment:
@sinpai so, if `current_user` isn't set, then `Balance:` text will be shown without number, it's a little bit confusing. What do you think about showing or not showing both `Balance:` and a number?

Reply:
Yes, it's a mistake here, for sure. I will fix this.